scripts/ci/tags.yaml

```diff
@@ -612,3 +611,49 @@ sample_edge_impulse:
     - nrf/lib/wave_gen/
     - nrf/samples/edge_impulse/
     - zephyr/soc/nordic/
+
+ci_applications_nrf_desktop:
+  files:
+    - bootloader/mcuboot/
+    - modules/crypto/mbedtls/
+    - modules/crypto/oberon-psa-crypto/
+    - modules/crypto/tinycrypt/
+    - modules/lib/open-amp/
+    - modules/lib/zcbor/
+    - nrf/applications/ipc_radio/
+    - nrf/applications/nrf_desktop/
+    - nrf/boards/nordic/
+    - nrf/drivers/mpsl/temp_nrf5/
```

Comment:
Can we make it: `nrf/drivers/mpsl` ? We also use e.g. `flash_sync` algorithm from here

Reply:
`nrf/cmake` and `nrf/sysbuild` should be added too I think (build system changes may break something too)

Reply:
> Can we make it: `nrf/drivers/mpsl` ? We also use e.g. `flash_sync` algorithm from here

Will add.

Reply:
> `nrf/cmake` and `nrf/sysbuild` should be added too I think (build system changes may break something too)

Indeed it may break something, but in this way all tests/samples should have dependency to this.
I prefer to not use this application to check build system behavior - its should be checked by all tests and samples and that should be force by the author of build system changes - by removing all optimization (similar like for upmerge).
Thus I am against adding such dependencies here.

Reply:
I am not sure about this one. Even a small change in Cmake files could lead to build failures. Let's discuss it briefly.